### ISSUE TITLE
User profile is not loading on store

### ISSUE THREAD

**Issue by houdiiny** [prescottprue/react-redux-firebase#654]
**Do you want to request a *feature* or report a *bug*?**  
Bug

**What is the current behavior?**
User profile is permanently at state: 
{isEmpty: true, isLoaded: false}
even with configurations set correctly:
    reduxFirestore(fbConfig),
    reactReduxFirebase(fbConfig, {
    	useFirestoreForProfile: true, 
    	userProfile: 'users-profile'
    })
The updateProfile on firebase works but profile is never loaded to the store
Permissions on firestore are:
    match /users-profile/{$uid} {
  		allow write: if isUserAdmin();
    	allow read: if true;
    }

**What is the expected behavior?**
User profile should load and display the fields set on firestore

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**
    "react": "^16.8.3",
    "react-dom": "^16.8.3",
    "react-materialize": "^2.6.0",
    "react-redux": "^5.1.1",
    "react-redux-firebase": "^2.2.6",
    "react-router-dom": "^4.3.1",
    "redux": "^4.0.1",
    "redux-firestore": "^0.6.4"

**Comment by prescottprue** [comment]
@houdiiny I am not able to reproduce locally, are you able to share a fully working example codebase (on codesandbox or similar). Here is a starting place: https://codesandbox.io/s/xpq3784jvw.

**Comment by houdiiny** [comment]
While trying to reproduce the problem on a "clean" simpler POC I stumbled across its solution, for some reason my client and backoffice stores were both being created and it generated some unexpected behavior (like not loading user profile on the client store). I rewrote some parts of it and now works correctly.

Thanks for [rubber ducking](https://en.wikipedia.org/wiki/Rubber_duck_debugging) this one for me.
